community/examples/flux-framework/flux-cluster.yaml

```diff
@@ -34,7 +34,7 @@ deployment_groups:
     settings:
       local_mount: /home
   - id: fluxfw-gcp
-    source: github.com/GoogleCloudPlatform/scientific-computing-examples.git/fluxfw-gcp/tf
+    source: github.com/GoogleCloudPlatform/scientific-computing-examples.git/fluxfw-gcp/tf?ref=867e558
```

Comment:
nit: Why not use the full hash?

Reply:
I've never seen the full hash used. I always use the output from

```
git rev-parse --short REF
```

where ref can be a hash or more generic like "HEAD". This command returns the commit hash shortened to the number of characters git believes ensures enough uniqueness given the size of the repository. In principle, this will break given enough commits but I wager it's a logarithmic number of commits we'd have to make.